src/reference/90-Developers-Guide/02-Coding-Guideline/00.md

```diff
@@ -175,13 +179,13 @@ object FooID {
 }
 ```
 
-#### Implicit converter for syntax (enrich-my-library) should be imported
+#### Implicit conversions for syntax (enrich-my-library pattern) should be imported
```

Comment:
Putting it in quotes leads some Google results, but substantially less. There is some precedent in fairly "official" sources though (https://www.playframework.com/documentation/2.2.x/ScalaJson) so if you'd like to revert it I don't feel strongly.